fastseq_cli/transformers_generate.py

```diff
@@ -546,7 +546,7 @@ def run_generate():
     parser.add_argument("--output_sequence_scores", action="store_true")
     parser.add_argument("--beam",
                         type=int,
-                        default=1,
+                        default=None,
                         required=False,
                         help="beam size for generation")
```

Comment:
Could we add more info here for `help="beam size for generation". Something like: help="beam size for generation. If None, beam size will be loaded from the model configuration file (the parameter name is num_beams). If the model configuration file does not have this parameter, beam size will be set as 1"